compiler/z/codegen/ConstantDataSnippet.cpp

```diff
@@ -498,6 +508,14 @@ TR::S390InterfaceCallDataSnippet::emitSnippetBody()
    *(intptrj_t *) cursor = 0;
    cursor += TR::Compiler->om.sizeofReferenceAddress();
 
+   // thunk to transition to the interpreter
+   // TODO: enable AOT once thunk relocation is supported.
```

Comment:
Do we have an issue we can link to in the code which tracks this work? If not can we open one and tag the appropriate people working on getting this done?